cypress/e2e/common/filters.cy.js

```diff
@@ -84,6 +88,15 @@ describe('Filter Panel', () => {
       cy.url().should('include', 'date_received_max=2020-10-31');
 
       cy.wait('@getComplaintsDateTo');
+
+      // check error handling and default values
+      cy.get('#date_received-from').type('2000-09-11', {
+        force: true,
```

Comment:
Is the `force: true` needed? Would `delay: 0` be an option?

Reply:
let me check this and get back to you

Reply:
Turns out it wasn't needed.  I cleaned up a bunch of the force true options